trymito.io/pages/excel-to-python/functions/math/ABS.tsx

```diff
@@ -0,0 +1,329 @@
+import GlossayHorizontalNavbar from '../../../../components/Glossary/HorizontalNav/HorizontalNav';
+import HorizontalNavItem from '../../../../components/Glossary/HorizontalNavItem/HorizontalNavItem';
+import CTAButtons from '../../../../components/CTAButtons/CTAButtons';
+import PageTOC from '../../../../components/Glossary/PageTOC/PageTOC';
+
+// Define the specific function name
+const functionName = "ABS";
+
+// Define your page content and metadata
+const PageContent = () => {
+
+  return (
+    <div className={excelToPythonStyles.body}>
+      <Head>
+        {/* Title Tag */}
+        <title>{`Excel to Python: ${functionName} - A Complete Guide`}</title>
+        
+        {/* Meta Description */}
+        <meta
+          name="description"
+          content={`Learn how to convert Excel's ${functionName} function to Python using Pandas. This comprehensive guide provides step-by-step instructions and practical examples.`}
+        />
+        
+        {/* Canonical URL (if applicable) */}
+        {/* <link rel="canonical" href={`https://www.example.com/excel-to-python/${function-name}-guide`} /> */}
+        
+        {/* Open Graph Tags (for social media sharing) */}
+        <meta
+          property="og:title"
+          content={`Excel to Python: ${functionName} function - A Complete Guide`}
+        />
+        <meta
+          property="og:description"
+          content={`Learn how to convert Excel's ${functionName} function to Python using Pandas. This comprehensive guide provides step-by-step instructions and practical examples.`}
+        />
+        {/* Add more Open Graph tags as needed */}
+        
+        {/* Twitter Card Tags (for Twitter sharing) */}
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta
+          name="twitter:title"
+          content={`Excel to Python: ${functionName} function - A Complete Guide`}
+        />
+        <meta
+          name="twitter:description"
+          content={`Learn how to convert Excel's ${functionName} function to Python using Pandas. This comprehensive guide provides step-by-step instructions and practical examples.`}
+        />
+        {/* Add more Twitter Card tags as needed */}
+        
+        {/* Other SEO-related tags (structured data, robots meta, etc.) */}
+        {/* Add other SEO-related tags here */}
+      </Head>
+      <Header />
+
+      <div className={pageStyles.container}>
+        <main className={classNames(excelToPythonStyles.main)}>
+          <div className={excelToPythonStyles.content}>
+            <div className={excelToPythonStyles.blog_content}>
+              <section className={classNames(excelToPythonStyles.title_card, excelToPythonStyles.section)}>
+                <div className={excelToPythonStyles.horizontal_navbar_container}>
+                  <GlossayHorizontalNavbar>
+                    <HorizontalNavItem title={'Function'} href={'/spreadsheet-automation'} />
```

Comment:
These Nav items will be updated once we build the `Function` and `Math` landing pages. But for testing purposes, this is fine